### DISCUSSION TITLE
SD card clone feature

### DISCUSSION THREAD

**Discussion by AdamF100001** [MichaIng/DietPi#6357] [Q&A]
Please can add an SD card clone feature?

Local clone or network features.

Thanks

**Comment by MichaIng** [comment]
Like `dd bs=1M if=/dev/mmcblk0 of=/path/to/elswehere`?

**Reply by AdamF100001** [reply]
Yes, but with a gui

**Reply by MichaIng** [reply]
Well, for such a single simple command a GUI is overkill, isn't it? Good to learn to use basic Linux tools for tasks which do not require a lot of commands or complex syntax. And the above really is pretty simple, isn't it?

**Reply by AdamF100001** [reply]
True, but a gui can be more accessible to new users.

**Reply by Joulinar** [reply]
but would not so experienced users run something like SD clone from command line??
